example/live_w_locator.html

```diff
@@ -34,10 +34,10 @@
                 <label>
                     <span>Barcode-Type</span>
                     <select name="decoder_readers">
-                        <option value="code_128" selected="selected">Code 128</option>
+                        <option value="code_128">Code 128</option>
                         <option value="code_39">Code 39</option>
                         <option value="code_39_vin">Code 39 VIN</option>
-                        <option value="ean">EAN</option>
+                        <option value="ean" selected="selected">EAN</option>
```

Comment:
Should not be the preferred option in the `master` branch. Please change that back